src/DependencyInjection/AnswearOverseasExtension.php

```diff
@@ -48,16 +47,4 @@ private function setLogger(ContainerBuilder $container, ?string $loggerClassName
             ]
         );
     }
-
-    private function validateConfig(array $config): void
-    {
-        //check if enum exists
-        EnvironmentEnum::get($config['environment']);
-
-        if (empty($config['apiKey']) || !is_string($config['apiKey'])) {
-            throw new \InvalidArgumentException(
-                'Provide valid apiKey config.'
-            );
-        }
-    }
```

Comment:
I changed the environment configuration to enumNode,
and gave up on strict apiKey checking - now it can also be an integer